rd-net/RdFramework.Reflection/ReflectionRdActivator.cs

```diff
@@ -148,7 +148,8 @@ private object ActivateRd(Type type)
       object instance;
       try
       {
-        instance = Activator.CreateInstance(implementingType);
+        instance = Activator.CreateInstance(implementingType) 
+                   ?? throw new InvalidOperationException($"Unable to create instance of: {implementingType.ToString(true)}");
```

Comment:
_(MInor)_ Sorry, why this change? It is not practical to expect `null` from `Activator.CreateInstance`: it will only return `null` if you pass a `Nullable` as the `implementingType`. It will throw an exception by itself otherwise; it doesn't need our additional assertion here.